terminus/tests/road_node_test.py

```diff
@@ -0,0 +1,15 @@
+import unittest
+
+from models.road_node import RoadNode
+
+from geometry.bounding_box import BoundingBox
+
+from geometry.point import Point
+
+
+class RoadNodeTest(unittest.TestCase):
+
+    def test_road_node_bounding_box(self):
```

Comment:
Probably a box with same origin and corner, I think there shouldn't be a problem with this. Ok, I'll write the test.